src/main/java/frc/robot/RobotContainer.java

```diff
@@ -61,13 +105,28 @@ public RobotContainer() {
   public Command getAutonomousCommand() {
     return m_autoCommand.getSelected();
   }
+  public void allianceChooserUpdate() {
+    alliance = DriverStation.getAlliance();
+    // alliance = m_allianceChooser.getSelected();
+    // Shuffleboard.getTab("Main").add("Alliance Chooser", m_allianceChooser);
+  }
```

Comment:
This practice seems odd.

You are calling this in disabledPeriodic. If the Alliance is needed, why not just call the DriverStation method when it is needed?

Reply:
I changed it to not have alliance stored in RobotContainer.